src/ptvsd/adapter/channels.py

```diff
@@ -65,11 +63,23 @@ def connect_to_ide(self, address=None):
             },
         )
 
-
     @singleton.autolocked_method
     def connect_to_server(self, address):
         assert self.server is None
-        raise NotImplementedError
+
+        # Import message handlers lazily to avoid circular imports.
+        from ptvsd.adapter import messages
+
+        host, port = address
+        sock = socket.create_client(host, port)
+        sock.connect(address)
+
+        server_stream = messaging.JsonIOStream.from_socket(sock, "SVR")
```

Comment:
I've moved off terse acronyms for these, now that we have a lot of channels all around :) Let's just do "server" etc.

The main reason why it was terse and all-caps before is because the messages were all logged as a single line, and so I wanted the channel name to stand out on that line, but without making it much longer. Now that timestamps are in their own column, and JSON is wrapped and indented, the first line of any message just has the channel name by itself - so it doesn't need caps to stand out, and it can be as verbose as we want.